pkg/clustersql/settings_test.go

```diff
@@ -143,6 +145,7 @@ func TestRangeMoveDuration(t *testing.T) {
 					{Target: "zone-1", Config: ZoneConfig{RangeMaxBytes: 20}},
 					{Target: "zone-2", Config: ZoneConfig{RangeMaxBytes: 30}},
 				},
+				// TODO: Is it required. Output will always be 0s because of interger division
```

Comment:
As discussed in Slack, we should open an issue here. This is passing because we have the same issue in the code under test. It might be the case that these perfect numbers don't exist in reality, but to be sure we should take a closer look.

For now, I think leaving these comments in place and adding an issue to track should be sufficient.